tests/test_routes.py

```diff
@@ -235,3 +242,80 @@ def test_route_sub_criteria_side_bar(
         assert (
             expected_names in response.data
         ), "Response does not contain expected name"
+
+    def test_route_sub_criteria_scoring_inaccessible_to_commenters(
+        self, flask_test_client
+    ):
+
+        # Mocking fsd-user-token cookie
+        test_payload = {
+            "accountId": "test-user",
+            "email": "test@example.com",
+            "fullName": "Test User",
+            "roles": ["COMMENTER"],
+        }
+        token = create_valid_token(test_payload)
+        flask_test_client.set_cookie("localhost", "fsd_user_token", token)
+
+        # Send a request to the route you want to test
+        response = flask_test_client.get(
+            "/assess/application_id/app_123/sub_criteria_id/1a2b3c4d?theme_id=score"  # noqa
+        )
+
+        # Assert that the response has the expected status code
+        assert 404 == response.status_code, (
+            "Commenter should receive a 404 when trying to access the sub"
+            " criteria scoring page"
+        )
+
+    @pytest.mark.parametrize(
+        "expected_ids, expected_names",
+        [
+            (b"general-information", b"General information"),
+            (b"activities", b"Activities"),
+            (b"partnerships", b"Partnerships"),
+        ],
+    )
+    def test_route_sub_criteria_side_bar_commenter(
+        self,
+        flask_test_client,
+        monkeypatch,
+        expected_ids,
+        expected_names,
+        mocker,
+    ):
+        mocker.patch(
+            "app.assess.models.ui.applicants_response.get_file_url",
+            return_value="sample1.doc",
+        )
+        # Mocking fsd-user-token cookie
+        test_payload = {
+            "accountId": "test-user",
+            "email": "test@example.com",
+            "fullName": "Test User",
+            "roles": ["COMMENTER"],
+        }
```

Comment:
Can we use the `test_commenter_claims` constant so we're not repeating this? 

https://github.com/communitiesuk/funding-service-design-assessment/blob/037a5d69c15a808a3e9a23186c0ebc73b49e2264/tests/conftest.py#L31-L36

Reply:
Yep done